grammars/css.cson

```diff
@@ -561,6 +561,44 @@
         'captures':
           '1':
             'name': 'punctuation.definition.entity.css'
+          '2':
+            'name': 'entity.other.namespace-prefix.css'
+          '3':
+            'name': 'punctuation.separator.css'
+          '4':
+            'name': 'entity.other.attribute-name.css'
+          '5':
+            'name': 'keyword.operator.pattern.css'
+          '6':
+            'name': 'string.unquoted.attribute-value.css'
+          '7':
+            'name': 'string.quoted.attribute-value.css'
+          '8':
+            'name': 'punctuation.definition.string.begin.css'
+          '9':
+            'name': 'punctuation.definition.string.end.css'
+          '10':
+            'name': 'punctuation.definition.entity.css'
+        'match': '''(?x)
+          (\\[)\\s*
+          (?:(\\g<ident>|\\*)?(\\|))?     # qualified name prefix
+          ([\\w:&&[^0-9]][\\w:.-]*+)\\s*  # attribute name
+          (?:
+            ([~|^$*]?=)\\s*
+            (?:                           # attribute value
+              (\\g<ident>)
+            | ((?>([\'"])(?:[^\\\\]|\\\\.)*?(\\k<-2>)))
+            )\\s*
+          )?
+          (\\])
+          (?<ident>-?+[a-zA-Z_\\\\\\P{ASCII}-][a-zA-Z0-9_\\\\\\P{ASCII}-]*+){0}
```

Comment:
Just to confirm, this isn't actually a matched line, correct?  You're just defining the ident capture group here?